benchmarks/README.md

```
@@ -7,7 +7,7 @@ See other files as examples.
 # Running
 
 ```
-$ npm start -w @typeberry/benchmark
+$ bun run --cwd packages/misc/benchmark start
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)